### ISSUE TITLE
dzVents does not get a port number when domoticz is started with sslwww port  and without www 

### ISSUE THREAD

**Issue by rwaaren** [domoticz/domoticz#4557]
in dzVents.cpp the current http port is requested with this call

`	luaTable.AddString("domoticz_listening_port", m_webservers.our_listener_port);`

This call returns an empty string when domoticz is started with somethin like 

`./domoticz -daemon -pidfile /var/run/domoticz.pid -www 0 -sslwww 8943
`

I tried to find the call to get the current https port but could not find one. 
Is such a call available and if so can you please enlighten me?

Thx

 [Forum post with reported issue](https://www.domoticz.com/forum/viewtopic.php?f=59&t=35056)

**Comment by gizmocuz** [comment]
It's a command line parameter or the default is used
It's in webserver_settings.listening_address (and) which is declared in domoticz.cpp

in the dzVents class you could declare at the top something like:

```
#include "../webserver/server_settings.hpp"

extern http::server::server_settings webserver_settings;
#ifdef WWW_ENABLE_SSL
extern http::server::ssl_server_settings secure_webserver_settings;
#endif

```

**Comment by rwaaren** [comment]
@gizmocuz. Thx for helping.

Fixed in #4560
